### DISCUSSION TITLE
Ensuring changelog is up to date

### DISCUSSION THREAD

**Discussion by cbkerr** [orgzly-revived/orgzly-android-revived#42] [Ideas]
I realized a few changes that didn't get a changelog entry. Can we add these posthoc?
* sync button in widget #12 
* fixes for searches with date ranges. #32, #33

When I was porting over PRs I didn't know where the changelog was generated from. We could use a PR template or checklist so authors can add it to the right place in the files.

**Comment by amberin** [comment]
This is on me. I put together the change log manually, and I missed those changes - sorry!

I couldn't find any signs of the previous change logs being auto-generated, and I wanted to stick to the previous praxis.

But I agree that a PR template or check list would be great. A user-friendly change log is needed.

**Comment by amberin** [comment]
I'm not sure how we can add them "post-hoc", but if anyone has an idea how to do it, I'm all for it.

In any case, we can always add these changes under the v1.8.11 heading when releasing v1.8.12.

**Comment by amberin** [comment]
>     * fixes for searches with date ranges. [Show ranged timestamps in agenda queries #32](https://github.com/orgzly-revived/orgzly-android-revived/pull/32), [Consider time and date ranges when querying events #33](https://github.com/orgzly-revived/orgzly-android-revived/pull/33)

This did not make it into 1.8.11, by the way. But "sync button in widget" did. I missed that one when writing the change log.
